Guardfile

```diff
@@ -24,7 +24,7 @@
 #  * zeus: 'zeus rspec' (requires the server to be started separately)
 #  * 'just' rspec: 'rspec'
 
-guard :rspec, cmd: "bundle exec rspec" do
```

Comment:
En fait c'est juste une application de ce qui est écrit littéralement trois lignes au dessus dans le même fichier :

> spring: 'bin/rspec' (This will use spring if running and you have installed the spring binstubs per the docs).